### PR TITLE
Allows pc to delete withdraw notes

### PR DESCRIPTION
**Pull request by purujitgoyal**

### CODE REVIEW

**Comment by melisabok** [comment]
thanks for the changes, I added a new [comment](https://github.com/openreview/openreview-py/issues/1294#issuecomment-1150046328) in the issue that says that only PCs can undo the withdrawal. To achieve this, remove the authors/signatures from the writers field. Only conference.id should be the writer of the note.

**Review by melisabok — APPROVED** [review]

File: openreview/conference/invitation.py [hunk 1]
```diff
@@ -459,12 +459,11 @@ def __init__(self, conference, note, reveal_submission):
                 'readers': readers,
                 'writers': {
                     'values-copied': [
-                        conference.get_id(),
-                        '{signatures}'
+                        conference.get_id(), conference.get_program_chairs_id()
```
Comment (carlosmondra) [inline openreview/conference/invitation.py:462]: This should no longer bee 'values-copied' it should be 'values'

File: tests/test_double_blind_conference.py [hunk 2]
```diff
@@ -1625,7 +1625,7 @@ def test_withdraw_submission(self, client, test_client, helpers):
             forum = notes[0].forum,
             replyto = notes[0].forum,
             readers = ['everyone'],
-            writers = [conference.id + '/Paper{number}/Authors'.format(number = notes[0].number), conference.id],
+            writers = [conference.id],
```
Comment (carlosmondra) [inline tests/test_double_blind_conference.py:1628]: This should match the invitation which is defined as:
`[ conference.get_id(), conference.get_program_chairs_id() ]`

**Review by celestemartinez — APPROVED** [review]
Works well 👍🏽

**Review by carlosmondra — APPROVED** [review]

**Pull request merged**